presto-parser/src/main/java/com/facebook/presto/sql/parser/SqlParserOptions.java

```diff
@@ -26,7 +26,7 @@
     //TODO: discuss - not the most extendible way of maintaining such a list.
     // Considerations: this list would change infrequently, and the warning is temporary.
     protected static final Set<String> RESERVED_WORDS_WARNING = ImmutableSet.of(
-            "CALLED", "called", "DETERMINISTIC", "deterministic", "FUNCTION", "function", "LANGUAGE", "language", "RETURN", "return", "RETURNS", "returns", "SQL", "sql");
+            "CALLED", "CURRENT_ROLE", "DETERMINISTIC", "FUNCTION", "LANGUAGE", "RETURN", "RETURNS", "SQL");
```

Comment:
Same here, most of this change belongs to the first commit (other than adding `CURRENT_ROLE`).